src/components/DeckMechanisms/index.js

```diff
@@ -468,6 +473,35 @@ export default class DeckMechanisms extends React.Component {
         break
       }
 
+      // Harvester of Souls
+      case 'N38': {
+        const id = arrayRandom(
+          cards
+            .filter(card => card.type === 'unit' && card.id !== 'T12')
+            .map(card => card.id)
+        )
+        const copiedCard = resolveCardForLevel({ id })
+        copiedCard.level = Math.floor(Math.random() * 5) + 1
+
+        copiedCard.weight = 0
+        copiedCard.id =
+          id +
+          '#' +
+          this.state.deck.filter(card => card.id.split('#')[0] === id).length
+
+        if (copiedCard.token) {
+          if (Math.random() < PROBABILITIES.NO_MOVEMENT_TOKEN) {
+            // Klaxi and Rain of Frogs spawn tokens with no movement
+            copiedCard.movement = 0
+          }
```

Comment:
Is that legitimately a thing? Mirz can spawns tokens with no movement?

Reply:
Not Mirz, but Harversters yes. It was discussed in #bugs-and-issues a few days ago

Reply:
That is incredibly bad. Fair enough, good catch.